app/javascript/App/bundles/common/components/CoinCharts/PriceGraph/chartOptions.js

```diff
@@ -1,3 +1,7 @@
+import _ from 'lodash'
+import moment from 'moment'
+import timeago from 'timeago.js'
```

Comment:
Do we need `timeago` if we already have `moment`? moment has a `fromNow()` that I think does the same thing https://momentjs.com/docs/#/displaying/fromnow/

Reply:
timeago was what was already being used in news
I'm fine with moving away from it though.